### ISSUE TITLE
Need for smarter lists

### ISSUE THREAD

**Issue by harisris** [YACReader/yacreader#3]
The purpose of importing tags from Comicvine is lost. There has to be smarter lists suggestions based on story arc and crossovers, and importing comics based on those into reader.

**Comment by selmf** [comment]
Yes, I agree. We definitely could make better use of the data we have.

The problem in this case is that the underlying code is pretty old and limited, but so tightly integrated with the modern stuff that we cannot implement the changes needed to get this to the next level without breaking a bunch of other things. We're currently working hard to get our codebase into a shape that will allow us to do exactly that, but this is a big task and we're not yet at the point where we are willing to rewrite that part of the project yet.

TL;DR: This is a known problem, we will improve it, but we can't say when it will happen …
